### DISCUSSION TITLE
Resident Evil 7: Biohazard

### DISCUSSION THREAD

**Discussion by Kermitsongs** [skyline-emu/skyline#1101] [Game Requests]
Residint evil 7

**Comment by lightyagamiklmn** [comment]
pretty sure it is cloud version and even that japan-only
so not possible to emulate

**Reply by Kermitsongs** [reply]
No I have computer better than from your mobile and I can download the game on it

**Comment by PixelyIon** [comment]
We will never support this as stated, it's the cloud version and it's Japan-exclusive. We don't want to risk connecting to Nintendo or Capcom servers so we will not support it.
